### ISSUE TITLE
Status dialog box hidden behind other threads

### ISSUE THREAD

**Issue by seckin206** [Reviewable/Reviewable#377]
<img width="853" alt="hidden_dialog" src="https://cloud.githubusercontent.com/assets/972671/17435243/d8f6b848-5b0f-11e6-9c36-616b22ef881f.PNG">

**Comment by pkaminski** [comment]
I can probably fix it if you can reproduce it...  @kballard [saw this before](https://github.com/Reviewable/Reviewable/issues/331) but we couldn't figure out how to make it happen consistently.

**Comment by seckin206** [comment]
Hopefully, I will try to reproduce it this evening.

**Comment by pkaminski** [comment]
Unable to reproduce so far?

**Comment by seckin206** [comment]
Sorry, this is lost in my schedule. Neither I or my colleagues were able to reproduce it. Should we close this one?

**Comment by pkaminski** [comment]
Let's keep it open with the `blocked` label.  I believe it's real, just very, very rare and hard to reproduce.  But having the issue up and visible might motivate the next person who runs into it to provide some more details.

**Comment by pkaminski** [comment]
Of course, less than a month after I closed this issue @nicolasartman successfully reproduced a layer order bug and I managed to fix it.  😛 Not sure if it's the same one as reported above, mind you...
